pkgs/racket-doc/scribblings/reference/eval-model.scrbl

```diff
@@ -946,8 +968,8 @@ In addition to the state that is shared among all threads, each thread
 has its own private state that is accessed through @deftech{thread
 cells}. A thread cell is similar to a normal mutable object, but a
 change to the value inside a thread cell is seen only when extracting
-a value from the cell from the same thread. A thread cell can be
-@deftech{preserved}; when a new thread is created, the creating
+a value from that cell in the same thread. A thread cell can be
+@deftech{preserved}: when a new thread is created, the creating
```

Comment:
+1 for keeping the `;`

Reply:
Do you mean I should revert to the semicolon? I'm not sure why.

Reply:
yes that's what I mean

I like using colons only to indicate that a list is coming, and semicolons for joining similar sentences.

no big deal if you disagree

Reply:
Colon can also mean "an explanation follows."